docs/src/rules/id-length.md

```diff
@@ -78,19 +78,19 @@ try {
 }
 var myObj = { apple: 1 };
 (num) => { num * num };
-function foo(num = 0) { }
+{ function foo(num = 0) { } }
 class MyClass { }
-class Foo { method() {} }
-class Foo { #method() {} }
-class Foo { field = 1 }
-class Foo { #field = 1 }
-function foo(...args) { }
-function foo([longName]) { }
+{ class Foo { method() {} } }
+{ class Foo { #method() {} } }
+{ class Foo { field = 1 } }
+{ class Foo { #field = 1 } }
+{ function foo(...args) { } }
+{ function foo([longName]) { } }
 var { prop } = {};
 var { prop: [longName] } = {};
 var [longName] = arr;
-function foo({ prop }) { }
-function foo({ a: prop }) { }
+{ function foo({ prop }) { } }
+{ function foo({ a: prop }) { } }
```

Comment:
I think the extra curlies here make the examples a bit confusing. Could we just rename the functions/classes to avoid collisions?